plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -1291,16 +1291,17 @@ public function maybe_init_wc_analytics() {
 		 * Hook: woocommerce_analytics_allow_tracking.
 		 *
 		 * @since 9.4.0
-		 * @param boolean $is_allowed Indicates if WooCommerce Analytics should be enabled.
+		 * @param boolean $is_allowed Indicates if WooCommerce Analytics is allowed to track.
 		 */
 		if ( ! apply_filters( 'woocommerce_analytics_allow_tracking', false ) ) {
 			return;
 		}
 
+		// Prevent to load the package for sites with Jetpack < 13.7. That version loads the package itself.
 		$jetpack_plugin = 'jetpack/jetpack.php';
 		if ( is_plugin_active( $jetpack_plugin ) ) {
 			$jetpack_plugin_data = get_plugin_data( WP_PLUGIN_DIR . '/' . $jetpack_plugin );
-			if ( ! isset( $jetpack_plugin_data['Version'] ) && version_compare( $jetpack_plugin_data['Version'], 'x.x.x', '<' ) ) {
+			if ( ! isset( $jetpack_plugin_data['Version'] ) && version_compare( $jetpack_plugin_data['Version'], '13.7', '<' ) ) {
```

Comment:
There's an issue with the logic here so we're always initialising WC Analytics regardless of what version of JetPack is installed.

```suggestion
			if ( isset( $jetpack_plugin_data['Version'] ) && version_compare( $jetpack_plugin_data['Version'], '13.7', '<' ) ) {
```